tinder.temple

```diff
@@ -0,0 +1,36 @@
+Tinder: project {
```

Comment:
Probably wouldn’t commit this, since wasn’t generated with it. Could put it in the readme?